### DISCUSSION TITLE
Support for RESP2

### DISCUSSION THREAD

**Discussion by zygisa** [valkey-io/valkey#985] [General]
Hi 👋 

We're using Redis with [Envoy Redis proxy](https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/other_protocols/redis.html) which currently doesn't support RESP3 so we're stuck with RESP2. I'm wondering if you have any plans to deprecate RESP2 in the near future (next major version, for example)?

**Comment by madolson** [comment]
No plans as of right now. Some features might be RESP3 only, especially since it provides some important functionality around out of band messages, such as better topology notifications during failovers. Any existing application should continue to work with RESP2 for the foreseeable future.
